scripts/update_builtin_functions.vim

```diff
@@ -0,0 +1,72 @@
+" create builtin function table
+
+" TODO more loose pattern?
+function! s:get_parse_lines(lines) abort
+  let from = index(a:lines, '} functions[] =')
+  if from ==# -1
+    return []
```

Comment:
```
throw 'cannot parse functions'
```

Reply:
agree. `update_builtin_commands.vim` also has the same code. thanks!

Reply:
fixed fc35a19c1eafb15d17e8fafbf2e6dfd7e0cc5e56